### PR TITLE
Replacing normalize by use of StandardScaler

### PR DESCRIPTION
**Pull request by KishenSharma6**
Normalize' will be removed in the version 1.2 release
of sklearn. Adding Pipeline with a StandardScaler as
a preprocessing stage.

#### Reference Issues/PRs
Fixes half of [#2143](https://github.com/alan-turing-institute/sktime/issues/2143)

#### What does this implement/fix? Explain your changes.

Normalize' will be removed in the version 1.2 release
of sklearn. Adding Pipeline with a StandardScaler as
a preprocessing stage.

#### What should a reviewer concentrate their feedback on?
Clarity of git commit message

##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).
- [ ] Optionally, I've updated sktime's [CODEOWNERS](https://github.com/alan-turing-institute/sktime/blob/main/CODEOWNERS) to receive notifications about future changes to these files.
- [ ] I've added unit tests and made sure they pass locally.

##### For new estimators
- [ ] I've added the estimator to the online documentation.
- [ ] I've updated the existing example notebooks or provided a new one to showcase how my estimator works.

### CODE REVIEW

**Comment by fkiraly** [comment]
Excellent, @KishenSharma6!
There are a couple linting errors (whitespace, newlines etc), kindly check the "details" in the code-quality test.
I strongly recommend you set up linters and possibly pre-commit locally (see the dev docs for that).

**Comment by KishenSharma6** [comment]
@fkiraly That was incredibly helpful feedback, thank you. 

I was able to set up pre-commit locally, I believe this is ready to be merged to the main branch. Please let me know if this requires further action, I will start on finishing the rest of https://github.com/alan-turing-institute/sktime/issues/2143

**Review by fkiraly — APPROVED** [review]
Looks good, thanks a lot!

**Comment by fkiraly** [comment]
Normalize' will be removed in the version 1.2 release
of sklearn. Adding Pipeline with a StandardScaler as
a preprocessing stage.

Fixes half of [#2143](https://github.com/alan-turing-institute/sktime/issues/2143)
